docs/source/newsfragments/4517.feature.rst

```diff
@@ -0,0 +1 @@
+:class:`.HierarchyObjectBase` and its inheritants now support :meth:`_get` for a dictionary-style handle retrieval (replaces deprecated :meth:`_id`), which returns :data:`None` on failure instead of raising an exception to aid with optional signal discovery.
```

Comment:
```suggestion
:class:`.HierarchyObjectBase` and its inheritants now support :meth:`_get` for a dictionary-style handle retrieval, which returns :data:`None` on failure instead of raising an exception to aid with optional signal discovery.
```
Do not speak of the devil (deprecated functionality). It makes it more difficult to remove all the references later and advertises it's existence, potentially leading to new users.

Reply:
haha ok